### ISSUE TITLE
can't delete error dats

### ISSUE THREAD

**Issue by juliangruber** [datproject/dat-desktop#400]

**Comment by juliangruber** [comment]
blocked by https://github.com/datproject/multidrive/pull/13

**Comment by Kriesse** [comment]
Maybe this is the same issue:
If I delete a dat from the desktop app, and also delete the folder via Finder, then refresh the app, the deleted app comes back. And the folder is created as well.

**Comment by juliangruber** [comment]
I think those are different issues. This is about adding a delete button to errored dats and fixing the issue in multidrive.

That's an interesting bug however! I'm opening a separate issue.

**Comment by juliangruber** [comment]
pull request here: https://github.com/datproject/dat-desktop/pull/405
